joblib/test/test_numpy_pickle.py

```diff
@@ -109,44 +106,16 @@ def _method(self):
 
 
 ###############################################################################
-# Test fixtures
-
-env = dict()
-
-
-def setup_module():
-    """ Test setup.
-    """
-    env['dir'] = mkdtemp()
-    env['filename'] = os.path.join(env['dir'], 'test.pkl')
-    print(80 * '_')
-    print('setup numpy_pickle')
-    print(80 * '_')
-
-
-def teardown_module():
-    """ Test teardown.
-    """
-    shutil.rmtree(env['dir'])
-    # del env['dir']
-    # del env['filename']
-    print(80 * '_')
-    print('teardown numpy_pickle')
-    print(80 * '_')
-
-
-###############################################################################
 # Tests
 
-def test_standard_types():
+def test_standard_types(tmpdir):
     # Test pickling and saving with standard types.
-    filename = env['filename']
+    filename = tmpdir.join('test.pkl').strpath
     for compress in [0, 1]:
         for member in typelist:
             # Change the file name to avoid side effects between tests
```

Comment:
Remove comment. Please check all the "Change the file name ..." there may be others lying around